charts/publisher/values.yaml

```diff
@@ -53,7 +53,7 @@ common:
     plekServicePublisherUri:
     plekServiceStaticUri:
     railsEnv: "production"
-    redisUrl: "redis://redis.ecs.test.govuk-internal.digital:6379"  # TODO: decide on Redis for EKS
+    redisUrl: "redis://redis"  # TODO: decide on Redis for EKS
```

Comment:
This is supposed to be just a generic example rather than the values specific to our AWS stack, which are currently in the argocd-apps chart. Hence just setting it to a plausible value that illustrates how to use it, rather than the specific hostname of one of our environments' Redises.